app/src/features/apiClient/screens/apiClient/components/views/components/response/TestsView/components/EmptyTestsView/EmptyTestsView.tsx

```diff
@@ -1,17 +1,64 @@
+import React, { useCallback } from "react";
 import { RQButton } from "lib/design-system-v2/components";
-import { MdRefresh } from "@react-icons/all-files/md/MdRefresh";
+import { MdAutoAwesome } from "@react-icons/all-files/md/MdAutoAwesome";
+import { MdArrowOutward } from "@react-icons/all-files/md/MdArrowOutward";
+import { useDeepLinkState } from "hooks";
+import { RequestTab } from "../../../../../http/components/HttpRequestTabs/HttpRequestTabs";
+
+interface EmptyTestsViewProps {
+  onGenerateTests?: () => void;
+  isGeneratingTests?: boolean;
+  canGenerateTests?: boolean;
+}
+
+export const EmptyTestsView: React.FC<EmptyTestsViewProps> = ({
+  onGenerateTests,
+  isGeneratingTests = false,
+  canGenerateTests = false,
+}) => {
+  const [, setDeepLinkState] = useDeepLinkState({ tab: RequestTab.QUERY_PARAMS });
```

Comment:
Let's avoid this, we can derive which tab to keep active from props and give that to `HttpRequestTabs`, same will be for graphql too.